src/Generator/TraitUsageGenerator.php

```diff
@@ -11,6 +11,12 @@
 use Reflection;
 use ReflectionMethod;
 
+/**
+ * Class TraitUsageGenerator
+ *
+ * @deprecated Deprecated in 2.7. Use UsageGenerator instead
```

Comment:
That is exactly why I believe t's name is too strict. The TraitUsageGenerator *is* being used to generate the use statements at the namespace level *in addition* to trait usage's.